.github/workflows/ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
       - name: Install python dependencies
         run: |
-          sudo python3 -m pip install molecule[docker] yamllint ansible ansible-lint docker openwisp-utils[qa]
```

Comment:
If I am not wrong, this will be required in the future though; direct use of pip will go away!

Reply:
ok we can revert this change!